checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
Hi Renato. Sorry for the delay. I had a series of deadlines last month but I have much more free time in the next week. I tried checker-maven-demo using the Maven compiler plugin using 3.3 against jdk8 and it worked great. However when I tried it against jdk7 it didn't work. Here's what I did:
In the pom.xml:
-Changed both instances of 1.8 to 1.7
-Changed both instances of jdk8 to jdk7
In the command line:
sudo update-alternatives --config javac
(I selected jdk8 javac as the default javac)
mvn clean
mvn compile

What am I missing?
Thanks.